maa-cli/src/main.rs

```diff
@@ -384,6 +398,16 @@ fn main() -> Result<()> {
         SubCommand::Complete { shell } => {
             generate(shell, &mut CLI::command(), "maa", &mut std::io::stdout());
         }
+        SubCommand::Copilot {
+            link,
+            disable_formation,
+            loop_times,
+        } => {
+            let _ = clean_trash(&proj_dirs);
+            copilot(&proj_dirs, link, disable_formation, loop_times);
+            let _ = run::run(&proj_dirs, ".copilot".to_owned(), None, false, false, false);
+            let _ = clean_trash(&proj_dirs);
```

Comment:
I don't think it's necessary to create a temporary file. Modifying `run::run` to accept either a file path or a TaskList seems like a better approach.

Reply:
`run::run` is high coupling, would it be better to split it into load resource, append task, and start?

Reply:
I completely agree, and it's on my priority list. However, I have other tasks to attend to, so it may take some time before I can address it.